moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h

```diff
@@ -64,6 +80,7 @@ struct MotionPlanDetailedResponse
   std::vector<std::string> description_;
   std::vector<double> processing_time_;
   moveit_msgs::msg::MoveItErrorCodes error_code_;
+  std::string planner_id_;
```

Comment:
Same comment here about this being a struct and not needing _ suffix.

Reply:
Sounds good, but this is requires changes in a lot of files so I'd rather prefer to make a follow up PR for this

Reply:
I opened an issue https://github.com/ros-planning/moveit2/issues/1689